src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -2401,6 +2401,11 @@ const bool Matcher::match_rule_supported_vector(int opcode, int vlen, BasicType
   } else { // NEON
     // Special cases
     switch (opcode) {
+    case Op_VectorReinterpret:
+      if (bit_size != 64 && bit_size != 128) {
```

Comment:
bit_size < 64 ?

Reply:
Reinterpret's size should bit_size == 64 ||  bit_size == 128